### PR TITLE
Centralized style.md and added PR template

### PR DESCRIPTION
**Pull request by sagersmith8**
### Changes
- Linked to centralized style.md
- Removed style.md
- Setup PR template

Please Review: @Kuroshii @sagesmith-wf

### CODE REVIEW

**Comment by sagersmith8** [comment]
[Link to centralized files and create PR Template for Vite Server](https://trello.com/c/Fr1fjizV/18-link-to-centralized-files-and-create-pr-template-for-vite-server)

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ This section contains information on setting up and running the project locally
 
 This section details information on contributing quality code to the repo
 
-- All code must adhere to the [style guide](STYLE.md)
+- All code must adhere to the [style guide](https://github.com/theVite/vite_server/blob/master/STYLE.md)
```
Comment (Kuroshii) [inline README.md]: Shouldn't this link to `vite_central`?

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ This section contains information on setting up and running the project locally
...
 - All code should be unit tested whenever possible. The goal is to have 70% code coverage repo wide. Test examples can be found in the [style guide](STYLE.md)
```
Comment (Kuroshii) [inline README.md]: And this?

File: README.md (hunk 1 above)
Comment (sagersmith8) [reply README.md]: Yea whoops

File: README.md (hunk 2 above)
Comment (sagersmith8) [reply README.md]: Same

**Review by Kuroshii — APPROVED** [review]
LGTM 👍

**Pull request merged**